sql/src/stream.cc

```diff
@@ -1438,6 +1454,10 @@ void stream::_process_instance(
   // Clean tables.
   _clean_tables(i.poller_id);
 
+  // Clean host/service commands caches
```

Comment:
This seems extreme. Every time a poller start or stop the *entire* command cache gets deleted whereas only entries related to the poller should be discarded.

Reply:
So what do you propose ?
A cache to store relations between hosts and instances ?
A cache that also contains the instance id ?
Another better idea?

Reply:
Yes, at the moment, we do not have any cache giving the relation between hosts and instances.
So I suggest to add such a map.

Reply:
I guess that a cache with the host / service relations with the instances would do it. Unfortunately I do not have any better idea.